physics/pascals_law.py

```diff
@@ -0,0 +1,59 @@
+"""
+Description :
+This law states that pressure exerted in some liquid which is at rest is the same in all the directions.
+OR
+Whenever an external pressure is applied on any part of a fluid contained in a vessel, it is transmitted undiminished and equally in all directions.
+
+Source: https://www.vedantu.com/physics/pascal-law
+Formulation: F = P * A
+Where, P= Pressure, F=Force and A=Area of contact
+
+Units:
+1 Newton = 1 Newton/(meters^2) X (meters^2)
+
+How to use?
+Inputs:
+    ___________________________________________________
+   |Name         | Units                   | Type      |
+   |-------------|-------------------------|-----------|
+   |pressure     | (in Newton/(meters^2))  | float     |
+   |-------------|-------------------------|-----------|
+   |area         | (in (meters^2))         | float     |
+   |_____________|_________________________|___________|
+
+Output:
+    ___________________________________________________
+   |Name         | Units                   | Type      |
+   |-------------|-------------------------|-----------|
+   |force        | (in Newtons)            | float     |
+   |_____________|_________________________|___________|
+

```

Comment:
```suggestion
```
This section is unnecessary. You already explain each of the variables and their units earlier in the comment.